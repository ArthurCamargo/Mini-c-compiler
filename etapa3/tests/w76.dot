digraph G {
n0;
n0 [label="f1"];

}