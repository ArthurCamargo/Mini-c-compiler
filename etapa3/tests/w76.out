0x55b324ce83c0
0x55b324ce83c0 [label="f1"];
